src/main/java/christmas/domain/promotion/discount/DiscountPromotion.java

```diff
@@ -0,0 +1,75 @@
+package christmas.domain.promotion.discount;
+
+import christmas.domain.consumer.Orders;
+import christmas.domain.consumer.VisitDay;
+import christmas.domain.promotion.badge.BadgePromotion;
+import christmas.domain.promotion.constants.PromotionCondition;
+
+import java.util.function.BiFunction;
+
+import static christmas.domain.consumer.constants.MenuCategory.DESSERT;
+import static christmas.domain.consumer.constants.MenuCategory.MAIN_DISH;
+import static christmas.domain.promotion.constants.PromotionCondition.*;
+
+public enum DiscountPromotion {
+    CHRISTMAS_D_DAY_DISCOUNT(
+            "크리스마스 디데이 할인",
+            CHRISTMAS_D_DAY_PROMOTION_CONDITION,
+            (visitDay, orders) -> visitDay.multiplyDate(100) + 900
```

Comment:
🤔**고민**🤔

저는 오히려 요구사항을 숨겼다고 생각했어요.
`1,000원으로 시작하여 크리스마스가 다가올수록 날마다 할인 금액이 100원씩 증가` 에서
1000원 (시작금액) 이라는 정보를 숨겼다고 생각됩니다.
비즈니스 로직을 다르게 표현하면 히스토리를 따라가기 어렵다고 생각이 드네요. 🥹

그렇다면 왜 이런 일이 생겼을까요? 🤔
매직 넘버(100, 900, 2023 등)에 이름을 붙이지 않아서 생긴 일이 아닐까 생각이 들어요.
요구 사항에 적힌 1,000 원이 시작 금액 (`START_DISCOUNT_AMOUNT`) 이라는 정보를 포함했으면 어땠을까 하는 생각이 듭니다.

만약에 기존 코드에서 900 이라는 매직넘버를 상수로 만들어 이름을 붙인다면 어떤 이름이 나올까요?
( 아래 `multiplyDate` 계산 로직도 변경되겠지만, 여기선 상수에 이름을 붙이는 것만 표현했습니다. )

```suggestion
            (visitDay, orders) -> START_DISCOUNT_AMOUNT + visitDay.multiplyDate(INCREASE_DISCOUNT_AMOUNT_BY_EVERY_DAY)
```